client/my-sites/themes/index.web.js

```diff
@@ -23,8 +23,11 @@ export default function( router ) {
 		if ( isLoggedIn ) {
 			router( '/design/:tier(free|premium)?', multiSite, navigation, siteSelection );
 			router( '/design/:tier(free|premium)?/:site_id', singleSite, navigation, siteSelection );
+			router( '/design/:tier(free|premium)?/filter/:filter', multiSite, navigation, siteSelection );
+			router( '/design/:tier(free|premium)?/filter/:filter/:site_id', singleSite, navigation, siteSelection );
```

Comment:
This is similar to the `/type/:tier` ones we had before, and unfortunately hard (if not impossible) to combine, since there aren't really and conditional route fragment helpers for two combined fragments.

Can we drop `/filter` if we make all `/:filter` options explicit, i.e. by something like

```js
const filters = [ 'color', 'style', 'whatever' ];
const filtersRoutePart = `/:filter(${ filters.join( '|' ) })`;
```

?